Library/Homebrew/cask/artifact/abstract_uninstall.rb

```diff
@@ -396,7 +396,10 @@ def uninstall_rmdir(*directories, command: nil, **_)
               command.run!("/bin/rm", args: ["-f", "--", ds_store], sudo: true, print_stderr: false)
             end
 
-            command.run("/bin/rmdir", args: ["--", resolved_path], sudo: true, print_stderr: false)
+            uninstall_rmdir(*resolved_path.children, command: command) if resolved_path.children.all?(&:directory?)
```

Comment:
What I had in mind was to check the return status of this call to `uninstall_rmdir` and only continue if this is successful.

Reply:
So the `return if directories.empty?` above should also be successful, i.e. change it to `return true if directories.empty?`.